EIPS/eip-7840.md

```diff
@@ -30,17 +31,20 @@ following shape:
 "blobSchedule": {
   "cancun": {
     "target": 3,
-    "max": 6
+    "max": 6,
+    "baseFeeUpdateFraction": 3338477
   },
   "prague": {
     "target": 6,
-    "max": 9
+    "max": 9,
+    "baseFeeUpdateFraction": 5007716
   }
 }
 ```
 
-When there is no explicit configuration for the current fork, use the last
-specified fork value. If no last value is specified, set both to zero.
+Clients **MUST** explicitly configure blob parameters in the configuration for Ethereum mainnet.
```

Comment:
Using "explicitly configure" and "explicit configuration" might lead to confusion, I think we are talking about slightly different things here. The point here is that the `blobSchedule` should be present in config (?) and it should be valid (the Cancun entry should be there)